Update AJS FAQs to include adding new context fields (#4102)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/troubleshooting.md

This works for any [context field](/docs/connections/spec/common/#context) that Segment automatically collects.

### Can I add context fields that do not already exist?

> Yes. Similar to overwriting context, you can add context fields by passing them into the options object as the third argument of the event call. For example, the analytics.js library does not automatically collect location information. So, if you wanted to add this into your context object, you would pass it into the third argument as in the example below:

Reviewer: ```suggestion
Yes. Similar to overwriting context, you can add context fields by passing them into the options object as the third argument of the event call. For example, the analytics.js library does not automatically collect location information. To add this into the context object, pass it into the third argument as in the example below:
```

```js
analytics.track("Order Completed", {}, {
    location: {
        latitude: '39.7392',
        longitude: '104.9903'
    }
})
```

> It is important to note that if you are not passing in properties, you still need to pass an empty object in it's place as shown in the above example. 

Reviewer: ```suggestion
> info ""
> You must pass the context object with the call, event if it's empty, as shown by the empty object in the example above.
```

> Additionally, custom context fields may not be forwarded to all destinations, as some destinations accept properties only. 

Reviewer: ```suggestion
Some destinations accept properties only. As a result, custom context fields you add may not forward to these destinations.
```
